instrumentation/grpc-1.6/src/main/java/io/opentelemetry/javaagent/instrumentation/hypertrace/grpc/v1_6/server/GrpcServerInterceptor.java

```diff
@@ -78,6 +78,32 @@ public <ReqT, RespT> ServerCall.Listener<ReqT> interceptCall(
     }
   }
 
+  /**
+   * Mapping according to https://github.com/grpc/grpc/blob/master/doc/http-grpc-status-mapping.md
+   */
+  private static Status mapHttpToGrpcStatus(int httpStatus) {
+    switch (httpStatus) {
+      case 400:
+        return Status.INTERNAL;
+      case 401:
+        return Status.UNAUTHENTICATED;
+      case 403:
+        return Status.PERMISSION_DENIED;
+      case 404:
+        return Status.UNIMPLEMENTED;
+      case 429:
+        return Status.UNAVAILABLE;
+      case 502:
+        return Status.UNAVAILABLE;
+      case 503:
```

Comment:
nit: should 5xx be mapped to INTERNAL ?

Reply:
No, We are using https://github.com/grpc/grpc/blob/master/doc/http-grpc-status-mapping.md as specified